app/controller/sdl/RCModulesController.js

```diff
@@ -874,8 +875,12 @@ SDL.RCModulesController = Em.Object.create({
                         this.climateModels[location_name].setClimateData(
                         data.params.moduleData.climateControlData);
                     if (Object.keys(data.params.moduleData.climateControlData).length > 0) {
-                        FFW.RC.onInteriorVehicleDataNotification({moduleType:'CLIMATE', moduleId: moduleUUId,
+                        prevData = sessionStorage.getItem('prevRCModuleData');
+                        sessionStorage.setItem('prevRCModuleData', JSON.stringify(data.params));
+                        if (prevData != JSON.stringify(data.params)) {
+                          FFW.RC.onInteriorVehicleDataNotification({moduleType:'CLIMATE', moduleId: moduleUUId,
                                                                 climateControlData: climateControlData});
+                        }
```

Comment:
@LiliiaShlikhtLuxoft this is a wrong fix. First of all, it solves the issue only for Climate module however it is applicable for all modules. Second, it is still sending a notification with the fields which were unchanged.
I think for this fix, you should go over functions like `setClimateData`, `setRadioData`, `setAudioControlData` ... and modify them to return only those fields which had different values before. In case if all values are still the same, you can return an empty object so HMI won't send a notification in that case.